### PR TITLE
Overhaul tutorials

### PR DESCRIPTION
**Pull request by cpollok**
New tutorials can be found here:
http://cram-system.org/tutorials/beginner/tut_review

When this PR goes through, the tutorials will be moved to the main site and replace the old ones.

### CODE REVIEW

File: cram_beginner_tutorial/package.xml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0"?>
 <package>
   <name>cram_beginner_tutorial</name>
-  <version>0.2.2</version>
+  <version>0.3.0</version>
   <description>The cram_beginner_tutorial package</description>
 
   <author>Lorenz Moesenlechner</author>
-  <author>Benjamin Brieber</author>
```
Comment (gaya-) [inline cram_beginner_tutorial/package.xml]: @cpollok Did you remove Benjamin because we no longer use his shape actions?

**Comment by gaya-** [comment]
@cpollok please add a bsd license tag with your name and year 2017 on every `.lisp` and `.asd` file.

**Comment by gaya-** [comment]
@cpollok I think we should switch to CRAM v0.4.0 as I would like to release it officially. We should discuss this the first thing when we meet next time.

File: cram_beginner_tutorial/package.xml (hunk 1 above)
Comment (cpollok) [reply cram_beginner_tutorial/package.xml]: Nope, I just failed at simple copy & paste. Added him back in.

**Pull request merged**